tslint.json

```diff
@@ -88,7 +88,8 @@
     // "radix": true,
     "semicolon": [
       true,
-      "always"
+      "always",
+      "ignore-bound-class-methods"
```

Comment:
If you have an arrow function bound to a class, prettier thinks you should have a semicolon on the end of it, and tslint does not. This causes chaos in any editor that uses both. This change will cause tslint to allow semicolons to follow arrow functions bound to classes.